app/containers/LiquidityPage/index.js

```diff
@@ -383,14 +374,15 @@ export function LiquidityPage(props) {
           {
             value: Web3.utils.toWei(EthValue.toString()),
             gasLimit: 1000000,
-            gasPrice: ethers.utils.parseUnits('10', 'gwei'),
+            gasPrice: ethers.utils.parseUnits('20', 'gwei'),
```

Comment:
Great job, i feel if the former gasPrice (10) doesn't affect anything it best we return it to it, also i feel changes on PRs should only address what the are meant to,

Reply:
Okay, noted.

Reply:
Done